compiler/src/iree/compiler/Codegen/LLVMCPU/test/set_transform_strategy.mlir

```diff
@@ -0,0 +1,64 @@
+// RUN: iree-opt --pass-pipeline='builtin.module(hal.executable(hal.executable.variant(iree-llvmcpu-lower-executable-target{test-lowering-configuration=true})))' --iree-codegen-llvmcpu-enable-transform-dialect-jit --split-input-file %s | FileCheck %s
+
+#pipeline_layout = #hal.pipeline.layout<push_constants = 0, sets = [
+  #hal.descriptor_set.layout<0, bindings = [
+    #hal.descriptor_set.binding<0, storage_buffer>,
+    #hal.descriptor_set.binding<1, storage_buffer>
+  ]>
+]>
+hal.executable private @pack  {
+  hal.executable.variant public @embedded_elf_x86_64, target = <"llvm-cpu", "embedded-elf-x86_64", {
+    cpu_features = "+avx512f,+avx2",
```

Comment:
Nice, this is something I was missing so far in my decision tree for GPUs, see my other comment on how to connect this in a retargetable way.